src/java/htsjdk/samtools/CRAMFileReader.java

```diff
@@ -272,6 +305,26 @@ public SAMRecordIterator assertSorted(final SortOrder sortOrder) {
         return it;
     }
 
+    CloseableIterator<SAMRecord> query(final int referenceIndex,
+                                       final int start, final int end, boolean overlap) throws IOException {
+        long[] filePointers = null;
+
+        // Hit the index to determine the chunk boundaries for the required data.
+        if (referenceIndex != -1) {
+            final BAMIndex fileIndex = getIndex();
+            final BAMFileSpan fileSpan = fileIndex.getSpanOverlapping(
+                    referenceIndex, start, -1);
+            filePointers = fileSpan != null ? fileSpan.toCoordinateArray()
+                    : null;
+        }
+
+        if (filePointers == null || filePointers.length == 0)
```

Comment:
either have no curly brackets and one line, or curly brackets and multiple lines.